src/OmniSharp.Roslyn.CSharp/Services/Navigation/FindSymbolsService.cs

```diff
@@ -30,57 +27,8 @@ public async Task<QuickFixResponse> Handle(FindSymbolsRequest request = null)
                 ? candidate.IsValidCompletionFor(request.Filter)
                 : true;
 
-            return await FindSymbols(isMatch);
+            return await _workspace.CurrentSolution.FindSymbols(isMatch,
+                p => !p.Name.EndsWith(".cake", StringComparison.OrdinalIgnoreCase));
```

Comment:
Could we try avoiding to use `.cake` extension in `OmniSharp.Roslyn.CSharp`, instead allow only `.cs` and `.csx`..? Would be difficult to get from project system, since `OmniSharp.Roslyn.CSharp` doesn't have (and shouldn't) have a dependency on `OmniSharp.Scripting`.

Reply:
Yeah, this feels weird. It would certainly fix the issue, but it feels a bit dirty and easily broken.

Reply:
Yeah I think being explicit about the file types supported makes sense, instead of excluding things.  If for example we ever add vb support, then this will start to leak again.